### DISCUSSION TITLE
What MacOS apps are you using?

### DISCUSSION THREAD

**Discussion by dlvhdr** [nicknisi/dotfiles#212] [General]
Just stumbled on your dotfiles repo, and wow there’s so much gold here! 

I was wondering which brew casks you have installed? Menu bar apps, GUI apps etc.

**Comment by nicknisi** [comment]
Thanks! I have a [Brewfile](https://github.com/nicknisi/dotfiles/blob/main/Brewfile) that lists all of the core items I install from homebrew.

As for other mac apps, I'll lis a few that I deem essential to my daily flow.

- [Obsidian](https://obsidian.md) is my go-to for note-taking. It's versatile, pluggable, and portable, and I absolutely love it
- [Cleanshot X](https://cleanshot.com) is something I used daily for capturing screnshots and videos for use in notes, Slack, and pull requests.
- [Audio Hijack](https://rogueamoeba.com/audiohijack/) and [Loopback](https://rogueamoeba.com/loopback/) are essential apps for managing and recording audio, which I use for the [JS Party](https://jsparty.fm) podcast.
- [Alfred](https://www.alfredapp.com/) is my go-to app launcher, but I'm also very interested in [Raycast](https://www.raycast.com/)
- Safari has been my daily driver browser for years and years, and I love it. However, I've been trying [Arc](https://arc.net/) and really like it, too!
- For front end development, I typically do most development in [Sizzy](https://sizzy.co/) (and of course, check other browsers)
- [OmniFocus](https://omnifocus.com) is my To-Do list app of choice on macOS and iOS
- [Kitty](https://sw.kovidgoyal.net/kitty/) is my terminal emulator
- [Hazel](https://www.noodlesoft.com/) is an app I use to automate moving files around
- I use [Keyboard Maestro](https://www.keyboardmaestro.com/main/) for automation, along with the Shortcuts app
- [Proxyman](https://proxyman.io/) is the debugging proxy I use for some development tasks and I really like it! It's super valuable when I need it.

This is a couple of the apps I use almost daily. I also have a [uses page](https://nicknisi.com/uses/) on my site that I try to keep regularly updated.

**Reply by dlvhdr** [reply]
Thank you!

Do you use anything for window management? Or is it some kind of alfred thing?

**Reply by nicknisi** [reply]
I'm constantly experimenting with this. Currently, I'm using [Moom](https://manytricks.com/moom/) but I'm also experimenting with [Amethyst](https://ianyh.com/amethyst/). I really like Amethyst but sometimes I feel like it's not flexible enough for my needs, juggling 3 large monitors.
